index.html

```diff
@@ -2128,6 +2228,216 @@ <h4>Bidirectional text</h4>
+        as long as their output behaviour is consistent with the provisions of this specification.</p>
+        <p>For example, a generic [[TTML2]] processor can process a TTML <a>document instance</a>
+        corresponding to the representation defined in <a href="#data-model"></a> without
+        performing such a mapping, especially for presentation purposes such as
+        rendering an audio mix based on the contents of the document.</p>
+      </div>
+      <p>Importantly, it is a design goal that if a DAPT processor supports the required features of this specification
+        then it should be able to process, within its supported feature set, a <a>DAPT document</a> for a future version of the specification.
+        Some future version could, for example, define some new grouping entity to encapsulate <a>Script Events</a>,
+        or allow some property to be specified on an entity where it is currently absent in the DAPT data model.</p>
+      <section class="informative">
+        <h3>Early identification of non-conformant documents</h3>
+        <p><em>Normative provisions relating to this section are defined in [[TTML2]].</em></p>
+        <p>Since it is a requirement of DAPT that <a>DAPT Documents</a> include a
+        <code>ttp:contentProfiles</code> attribute on the root element,
+        and that the attribute includes a value equal to the <a>DAPT 1.0 Content Profile</a>,
+        as specified at <a href="#ttp-contentprofiles"></a>,
+        it follows that any TTML document that does not include such an attribute can be considered
+        not to be a <a>DAPT Document</a>;
+        therefore a processor requiring strict adherence to DAPT could stop processing
+        such a document.</p>
+      </section>
+      <section>
+        <h3>Document conformance claims on output</h3>
+        <p>If a processor takes as its input a <a>DAPT document</a> that claims conformance to
+          one or more content profiles which that processor does not support,
+          allows the document to be modified, and then outputs a document based on that
+          modified source, <a href="#ttp-contentprofiles"></a> prohibits it from including in <code>ttp:contentProfiles</code>
+          the values of those content profiles which it does not support.
+        </p>
+        <p>Conversely, a processor that takes as its input a <a>DAPT document</a> that
+          contains vocabulary relating to features that it does support,
+          but where support for those features is excluded from the content profiles
+          to which the document claims conformance, SHOULD NOT implement
+          those features in the context of the document, and MAY take
+          additional validation steps to check the semantics of the document.</p>
+      </section>
+      <section>
+        <h3>Handling <code>&lt;div&gt;</code> and  <code>&lt;p&gt;</code> elements</h3>
```

Comment:
I feel that we are introducing this section 6 (TTML to model) as a convoluted way to work around the restrictions we put in section 4 (model to TTML). I wonder if we should not just relax section 4, specifically 4.1.4 saying that the list of script events is represented by a set of structures each representing a Script Event (as defined in 4.3) but that structure may be a direct child of the body or nested in div elements.

Reply:
That was certainly one of the options we discussed, defining a meaning for `div` elements that contain other `div` elements, as a grouping mechanism, but I thought you wanted something simpler and more tightly defined with less room for flexibility?

I wouldn't want to blandly permit nested `div`s without some explanation of what they represent: it doesn't feel right for DAPT, to me, but I'm open to persuasion.